CMakeLists.txt

```diff
@@ -291,14 +254,13 @@ function(create_library)
     target_link_libraries(${PROJECT_NAME} PUBLIC OpenMP::OpenMP_CXX)
   endif()
 
-  target_link_libraries(${PROJECT_NAME} PUBLIC proxsuite-nlp::proxsuite-nlp)
-  target_link_libraries(${PROJECT_NAME} PUBLIC Boost::boost)
   target_link_libraries(${PROJECT_NAME} PUBLIC fmt::fmt)
-  # set the install-tree include dirs
-  # used by dependent projects to consume this target
+  target_link_libraries(${PROJECT_NAME} PRIVATE Tracy::TracyClient)
```

Comment:
We should check that tracy symbol in aligator.so are not public.

Reply:
Fixed it using your work on qhull+hppfcl :)